src/js/core/services/ui-grid-util.js

```diff
@@ -1225,7 +1225,7 @@ module.service('gridUtil', ['$log', '$window', '$document', '$http', '$templateC
     function runFunc(endDate){
       lastCall = +new Date();
       func.apply(context, args);
-      $interval(function(){ queued = null; }, 0, 1);
+      $interval(function(){ queued = null; }, 0, 1, false);
```

Comment:
If the goal of this is to remove the reference to queued shouldn't this be calling `$interval.cancel(queued)`

Perhaps I don't fully understand how this function works.